org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/IBuildSupport.java

```diff
@@ -137,4 +137,10 @@ default void registerPreferencesChangeListener(PreferenceManager preferenceManag
 	default void unregisterPreferencesChangeListener(PreferenceManager preferenceManager) throws CoreException {
 	}
 
+	String buildToolName();
```

Comment:
You should make it as default method to not break other build support implementors from other extensions.

Reply:
@testforstephen OK, thanks for advice!